### PR TITLE
2015 10 cc disabled textareas

### PR DESCRIPTION
**Pull request by local-girl**
as requested by #1750. Stops textareas from resizing when disabled. They still look the same however, to discuss with designers as this is an input-wide concern.

### CODE REVIEW

File: src/adhocracy_frontend/adhocracy_frontend/static/stylesheets/scss/components/_forms.scss [hunk 1]
```diff
@@ -59,6 +59,21 @@ They should all look and behave similarily.
...
+parent: textinput
+---
+
+Textareas behave exactly like text inputs however they are resizeable when enabled in modern browsers.
+
+```html_example
+<textarea></textarea>
+<textarea disabled="disabled" placeholder="disabled"></textarea>
+```
+*/
```
Comment (xi) [inline src/adhocracy_frontend/adhocracy_frontend/static/stylesheets/scss/components/_forms.scss]: There is already a textarea in the textinput documentation. That should probably be removed.

**Comment by xi** [comment]
There is still the issue that disabled textareas are not visually distinguishable from non-disabled ones.

File: src/adhocracy_frontend/adhocracy_frontend/static/stylesheets/scss/components/_forms.scss (hunk 1 above)
Comment (local-girl) [inline src/adhocracy_frontend/adhocracy_frontend/static/stylesheets/scss/components/_forms.scss]: But that's true of all inputs, the designers need to think about this and it should be a separate pull request I think

**Comment by local-girl** [comment]
But that's true of all inputs, the designers need to think about this and it should be a separate pull request I think
